app/questionPage.html

```diff
@@ -73,16 +53,40 @@ <h3 id="timer"></h3>
                 <div class="question-container col-md-8">
                     <div id="questions"></div>
                     <div id="question-nav" class="question-nav"></div>
-                    <button type="button" id="finish" class="btn btn-success finish">Finish</button>
                 </div>
-                <div class="col-md-2">
+                <div class="col-md-2 overview_column">
                     <button type="button" class="btn btn-info next">Next</button>
+                    <button type="button" class="btn btn-info overview">Overview</button>
```

Comment:
It looks a little odd have this appear out of line with the previous button. Perhaps consider moving this to the middle and below the question container, where the finish button used to be?

Reply:
Solved in a8d11d6